scipy/stats/_mstats_extras.py

```diff
@@ -55,6 +55,28 @@ def hdquantiles(data, prob=list([.25,.5,.75]), axis=None, var=False,):
     --------
     hdquantiles_sd
 
+    Examples
+    --------
+    >>> import numpy as np
+    >>> from scipy.stats.mstats import hdquantiles
+
+    >>> # Sample data
+    >>> data = np.array([1.2, 2.5, 3.7, 4.0, 5.1, 6.3, 7.0, 8.2, 9.4])
+
+    >>> # Define quantiles to compute
+    >>> quantiles = [0.25, 0.5, 0.75]
```

Comment:
Since these numbers are not the quantiles:
```suggestion
    >>> probabilities = [0.25, 0.5, 0.75]
```
Or similar, also needs to be changed below.